.idea/checkstyle-idea.xml

```diff
@@ -3,17 +3,17 @@
   <component name="CheckStyle-IDEA">
     <option name="configuration">
       <map>
-        <entry key="active-configuration" value="PROJECT_RELATIVE:$PROJECT_DIR$/checkstyle_ide.xml:Elasticsearch" />
-        <entry key="checkstyle-version" value="8.42" />
+        <entry key="active-configuration" value="PROJECT_RELATIVE:/Users/rory/src/elastic/elasticsearch/checkstyle_ide.xml:Elasticsearch" />
```

Comment:
Looks like IntelliJ snuck this change in. We should still be using `$PROJECT_DIR$` here.